.github/workflows/ci.yml

```diff
@@ -47,5 +49,40 @@ jobs:
           pip install --upgrade .
 
       - name: Running checks ☑
+        id: precommit
         if: matrix.python-version != '3.11'  # Pylint currently triggers erroneous no-member errors
-        run: pre-commit run --all-files --verbose
+        uses: mathiasvr/command-output@v2.0.0
```

Comment:
Why introducing a dependency to another GA workflow script?

Unless it's necesseray or really useful, I'd prefer to limit the number of dependencies.

Having many such dependencies statistically increases the risk of of them breaking in the future, causing some failures & maintenance costs (time to identify & fix the problem) to this project.